### ISSUE TITLE
Accessibility page has lists, but they are not marked up as lists

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#1810]
## Description of issue

On the accessibility page, we have a list in a table in a disclosure component. The list is not an actual list, but has hard-coded hyphens, suggesting it should be a list.

Through GC Articles, it seems we cannot put lists inside of table cells. 

Add fix:
<img width="2000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBd1ZRQ1E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--8593a0015400db47e1b7f16e9224d40350c26212/Capture%20d%E2%80%99e%CC%81cran%2C%20le%202025-03-18%20a%CC%80%2015.32.21.png" alt="Capture d’écran, le 2025-03-18 à 15.32.21.png" />

Second fix:
<img width="1038" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNUVkQ3c9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--19fbd9316b2a725e22334d33ee74e5c3d0f957b0/image.png" alt="image.png" />

## Potential fix

- Remove the need for lists inside table cells
- Move the a11y page away from GCA so we can control content markup more finely, though less easily.

## Resources

- Links to documentation or articles that could be helpful
- examples: WCAG docs, ARIA docis, web.dev, Deque, Webaim...

## Acceptance Criteria

- [ ] Should this issue be added to the A11y Statement if it cannot be resolved soon?

**Comment by yaelberger-commits** [comment]
Yael to QA page for A11y statement for bulleted list

**Comment by yaelberger-commits** [comment]
@andrewleith it seems there are still some issues with the lists not being bulleted as lists in the FR page of the A11y statement. Here are some screenshots 

<img width="1210" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNDBoQ3c9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--9c6ed7fe6cebdf8cb3908ff3695bce18397a7889/Screenshot%202025-08-21%20at%208.18.59%E2%80%AFAM.png" alt="Screenshot 2025-08-21 at 8.18.59 AM.png" />
<img width="1180" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNDRoQ3c9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--beb9e5f331666e61f74b1899e9ff359323882f5d/Screenshot%202025-08-21%20at%208.19.08%E2%80%AFAM.png" alt="Screenshot 2025-08-21 at 8.19.08 AM.png" />
